api-tests/features/p2-f2f-journey.feature

```diff
@@ -138,3 +138,68 @@ Feature: P2 F2F journey
       # Initial journey
       When I get an 'temporarily_unavailable' OAuth error from the CRI stub
       Then I get a 'pyi-technical' page response
+
+    Scenario Outline: Async queue error
+      When I get an error from the async CRI stub
+      Then I get a 'page-face-to-face-handoff' page response
+
+      # Return journey
+      When I start a new 'medium-confidence' journey and return to a 'pyi-f2f-technical' page response
+      When I submit a 'end' event
+      Then I get an OAuth response
+
+  Scenario: f2f PYI escpae route
+    Given I start a new 'medium-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'end' event
+      Then I get a 'page-ipv-identity-postoffice-start' page response
+      When I submit an 'end' event
+      Then I get a 'pyi-escape' page response
+
+  Rule: F2f evidence requested strength score
+    Background: User has pending f2f verification
+      Given I start a new 'medium-confidence' journey
+      When I submit an 'appTriage' event
+      Then I get a 'dcmaw' CRI response
+      When I get an 'access_denied' OAuth error from the CRI stub
+      Then I get a 'page-multiple-doc-check' page response
+      When I submit a 'end' event
+      Then I get a 'pyi-post-office' page response
+      When I submit a 'next' event
+      Then I get a 'claimedIdentity' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get an 'address' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get a 'fraud' CRI response
+
+    Scenario Outline: requested strengh score three for fraud score 2
```

Comment:
Why are these `Scenario Outline`s - if they're not parameterised, should they just be `Scenario`s?

(Alternatively, they are quite repetitive and maybe could be combined a single outline)